Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/SortAdapter.cpp

```diff
@@ -38,6 +39,60 @@ namespace AZ::DocumentPropertyEditor
         }
     }
 
+    Dom::Path RowSortAdapter::MapPath(const Dom::Path& path, bool mapToSource)
```

Comment:
comment: Not a huge fan of `bool` params as the callsites can be cryptic and the complexity of the function immediately increases. Consider at least making this an enum if possible (you can ignore but bool params are just the worst 😅 I'm now convinced of it 😝)